src/lib/Icons.ts

```diff
@@ -234,6 +237,7 @@ const icons: { [key in IconName]: { light: IconDefinition; solid: IconDefinition
   hide: { light: faEyeSlashLight, solid: faEyeSlashSolid },
   'image-gallery': { light: faImagesLight, solid: faImagesSolid },
   info: { light: faInfoCircleLight, solid: faInfoCircleSolid },
+  'info-square': { light: faInfoSquareLight, solid: faInfoSquareSolid },
```

Comment:
Ik weet nu niet of Karel hier nou iets over had gezegd, maar nu hebben we twee info iconen. We zouden dat natuurlijk niet willen. Zou je dit misschien ook in ObsIdentify willen hebben?

Reply:
Niet meer nodig, dus weer verwijderd.

Reply:
👍